Source/WebKit/NetworkProcess/NetworkResourceLoader.cpp

```diff
@@ -1000,7 +1000,7 @@ void NetworkResourceLoader::didReceiveResponse(ResourceResponse&& receivedRespon
 
 void NetworkResourceLoader::sendDidReceiveResponsePotentiallyInNewBrowsingContextGroup(const WebCore::ResourceResponse& response, PrivateRelayed privateRelayed, bool needsContinueDidReceiveResponseMessage)
 {
-    auto browsingContextGroupSwitchDecision = toBrowsingContextGroupSwitchDecision(m_currentCoopEnforcementResult);
+    auto browsingContextGroupSwitchDecision = m_connection->networkProcess().usesSingleWebProcess()? BrowsingContextGroupSwitchDecision::StayInGroup: toBrowsingContextGroupSwitchDecision(m_currentCoopEnforcementResult);
```

Comment:
I'm surprised the style checker didn't catch this one, but:
```suggestion
    auto browsingContextGroupSwitchDecision = m_connection->networkProcess().usesSingleWebProcess() ? BrowsingContextGroupSwitchDecision::StayInGroup : toBrowsingContextGroupSwitchDecision(m_currentCoopEnforcementResult);
```